metadata-ingestion/src/datahub/ingestion/source/common/subtypes.py

```diff
@@ -0,0 +1,40 @@
+from enum import Enum
+
+
+class DatasetSubTypes(str, Enum):
+    # Generic SubTypes
+    TABLE = "Table"
+    VIEW = "View"
+    TOPIC = "Topic"
+    SCHEMA = "Schema"
```

Comment:
is this one used for datasets? i thought it would only be needed for containers

Reply:
yes, because we support representation of "schemas" as in Protobuf / Json Schemas as sub-types of Datasets currently.

Reply:
ah got it